cmd/multitenant-autoscaler/main.go

```diff
@@ -79,12 +84,22 @@ func main() {
 		logger.Fatal("Error building kubernetes clientset.", zap.Error(err))
 	}
 
+	rm := restmapper.NewDeferredDiscoveryRESTMapper(cached.NewMemCacheClient(kubeClientSet.Discovery()))
+	go wait.Until(func() {
+		rm.Reset()
+	}, 30*time.Second, stopCh)
+	scaleKindResolver := scale.NewDiscoveryScaleKindResolver(kubeClientSet.Discovery())
+	scaleClient, err := scale.NewForConfig(cfg, rm, dynamic.LegacyAPIPathResolverFunc, scaleKindResolver)
+	if err != nil {
+		logger.Fatal("Error building kubernetes clientset.", zap.Error(err))
```

Comment:
Shouldn't this log mention the scale client rather than the kube client?

Reply:
Done